compiler/resolution/AutoDestroyScope.h

```diff
@@ -44,6 +44,9 @@ class AutoDestroyScope {
 
   void                     addEarlyDeinit(VarSymbol* var);
 
+  // Counts how many declarations exists
```

Comment:
nit, "Counts how many declarations exist"

Reply:
This comment could be better even setting aside the typo. I'll touch it up